charts/common/templates/ingress-cloudarmor.yaml

```diff
@@ -1,5 +1,49 @@
 {{- if .Values.cloudArmor.enabled -}}
 {{- $svcport := .Values.service.port -}}
+{{- if hasKey .Values.cloudArmor "hosts" -}}
+{{- $commonname := include "common.name" . -}}
+{{- $labels := include "common.labels" . -}}
+{{- $secretname := .Values.cloudArmor.certificate.secretName | default (include "common.cloudArmorTlsSecret" .) -}}
+{{- $staticipname := .Values.cloudArmor.staticIpName | default (include "common.name" .) -}}
+{{- range .Values.cloudArmor.hosts }}
+{{- $hostwithdashes := printf "%s-ca" .host | replace "." "-" -}}
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ .name | default $hostwithdashes }}
```

Comment:
Can we use the same naming convention on this one as the original resource `{{ include "common.cloudArmorName" . }}`